README.md

```diff
@@ -1,66 +1,55 @@
-# APP title
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> APP description...
+# Decode Morse Code
```

Comment:
## OPTIONAL
- Please remove " Decode Morse code" and Change it to `Enumerable` because it is the name of the Project.

Reply:
Thank you @OLIPLICHE. The issue has been resolved on https://github.com/Tadesse-Alemayehu/ruby-Enumerable/pull/1/commits/fe7e8e89431db616a73ea2fc1fcb733d8e11dfa9 commit hash.

Reply:
Well resolved 👍🏽